package.json

```diff
@@ -32,5 +45,13 @@
         "queries/highlights.scm"
       ]
     }
+  ],
+  "files": [
+    "grammar.js",
+    "binding.gyp",
+    "prebuilds/**",
+    "bindings/node/*",
+    "queries/*",
+    "src/**"
```

Comment:
The rest of this file had automated changes by.. I'm not sure what exactly. I'm not too familiar with node development and so I was a little surprised to see anything in `package.json` get changed apart from those 2 lines I edited.